src/main/resources/avro/biodata.avdl

```diff
@@ -0,0 +1,133 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines the "biodata" objects, which can be considered data
+representations of biological correlates.
+*/
+
+protocol Biodata {
+
+  import idl "common.avdl";
+  import idl "metadata.avdl";
+
+ /**
+  An individual (or subject) typically corresponds to an individual
+  human or other organism.
+  */
+  record Individual {
+
+    /**
+    The Individual's :ref:`id <apidesign_object_ids>`. This is unique in the
+    context of the server instance.
+    */
+    string id;
+
+    /**
+    The Individual's :ref:`name <apidesign_object_names>`. This is a label or
+    symbolic identifier for the individual.
+    */
+    union { null, string } name = null;
+
+    /**
+    The Individual's description. This attribute contains human readable text.
+    The "description" attributes should not contain any structured data.
+    */
+    union { null, string } description = null;
+
+    /**
+    The :ref:`ISO 8601<metadata_date_time> time at which this Individual record
+    was created.
+    */
+    string createDateTime;
+
+    /**
+    The :ref:`ISO 8601<metadata_date_time> time at which this Individual record
+    was updated.
+    */
+    string updateDateTime;
+
+    /**
+    For a representation of an NCBI Taxon ID as an OntologyTerm, see
+    NCBITaxon Ontology
+      http://www.obofoundry.org/ontology/ncbitaxon.html
+    For example, 'Homo sapiens' has the ID 9606. The NCBITaxon ontology ID for
+    this is NCBITaxon:9606, which has the URI
+    http://purl.obolibrary.org/obo/NCBITaxon_9606
+    */
+    union { null, OntologyTerm } species = null;
```

Comment:
This is different to Reference which requires an ID (e.g. 9606->human).  Is the intention to update Reference?